manager/middlewares/authentication.go

```diff
@@ -0,0 +1,40 @@
+package middlewares
+
+import (
+	"app/base/utils"
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+const KEY_ACCOUNT = "account"
+
+
+type ErrorResponse struct {
```

Comment:
This struct already exists in controllers.structures. So what to move into base.utils.gin and use it in both places?

Reply:
Thanks, fixed.